O3-1.tex

```diff
@@ -22,37 +22,40 @@
 
 \title{The LSST Data Management System}
 \author{
-Mario~Juric,$^1$
-Jeffrey~Kantor,$^2$
```

Comment:
These commas were in the right place. The example latex file distributed by the conference committee has the commas before the superscript.